Automattic/jetpack
# Fix social previews crashing for non admin authors

Fixes https://github.com/Automattic/jpop-issues/issues/8952

Social Previews uses `core` store to get the site title and `getEntityRecord( 'root', 'site' )` doesn't work for non-admins, thus it crashes when `.title` is read from that.

Related:
- https://github.com/WordPress/gutenberg/pull/45093
- https://github.com/WordPress/gutenberg/issues/33003

## Proposed changes:

* Use the `getUnstableBase` selector to get the readonly site settings which are available for unauthenticated users.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Using `trunk`, create a non-admin author account
* Login as that new author account
* Create a post
* Open the Jetpack or Social Sidebar
* Open Social Previews
* Goto Google or Mastodon previews
* It should crash
* Now boot up this PR
* Do the above steps
* Confirm that it does not crash
* Confirm that the site title works fine for Google and Mastodon
* Confirm that the above works fine for admin users as well
